portfolio/src/main/webapp/script.js

```diff
@@ -90,4 +93,16 @@ function fetchBlobstoreUrl() {
         const messageForm = document.getElementById('my-form');
         messageForm.action = imageUploadUrl;
       });
+}
+
+async function checkLogIn(){
+    const formContainer = document.getElementById('my-form');
+    formContainer.style.display = "none";
+    const response = await fetch('/LogInServlet');
+    const text = await response.text(); // Text is body tags with 404 errors
+    if(text != "null"){
```

Comment:
are you meaning to check for `null` or `undefined` value here? You are checking that the text is literally the string `'null'`

Probably, a "falsy" check is what we want here

```javascript
if (text) {
  // text is any value except null, undefined, 0, NaN, false, ""
  // yeah javascript is weird
}
```
https://developer.mozilla.org/en-US/docs/Glossary/Falsy

Reply:
I ended up returning an object instead. This was an inefficient way of going about things